CMakeLists.txt

```diff
@@ -34,40 +33,36 @@ add_library(nalu "")
 add_executable(${nalu_ex_name} ${CMAKE_CURRENT_SOURCE_DIR}/nalu.C)
 add_executable(${utest_ex_name} ${CMAKE_CURRENT_SOURCE_DIR}/unit_tests.C)
 
+########################## MPI ####################################
 find_package(MPI REQUIRED)
-
-if(ENABLE_OPENMP)
-  find_package(OpenMP)
-  separate_arguments(OpenMP_CXX_FLAGS)
-  separate_arguments(OpenMP_Fortran_FLAGS)
-  target_compile_options(nalu PUBLIC $<$<COMPILE_LANGUAGE:CXX>:${OpenMP_CXX_FLAGS}>)
-  target_compile_options(nalu PUBLIC $<$<COMPILE_LANGUAGE:Fortran>:${OpenMP_Fortran_FLAGS}>)
-endif()
+target_link_libraries(nalu PUBLIC $<$<BOOL:${MPI_CXX_FOUND}>:MPI::MPI_CXX>)
+target_link_libraries(nalu PUBLIC $<$<BOOL:${MPI_Fortran_FOUND}>:MPI::MPI_Fortran>)
 
 ########################## TRILINOS ####################################
+set(USER_Trilinos_DIR ${Trilinos_DIR})
 set(CMAKE_PREFIX_PATH ${Trilinos_DIR} ${CMAKE_PREFIX_PATH})
-find_package(Trilinos REQUIRED)
+find_package(Trilinos QUIET REQUIRED)
 if(Trilinos_FOUND)
-  message(STATUS "Found Trilinos = ${Trilinos_DIR}")
+  message(STATUS "Found Trilinos = ${USER_Trilinos_DIR}")
```

Comment:
How can we be certain that `find_package` settled on the Trilinos in `Trilinos_DIR`? It seems that we are just depending on the fact that we pushed it to the top of `CMAKE_PREFIX_PATH` stack.

Reply:
That's true. I didn't like it reporting the path with `lib/cmake` in it and overwriting `Trilinos_DIR`. The only way I could figure out how to avoid it was to report the path by saving the original. The only way to get consistent directories for the user seems to be by using `CMAKE_PREFIX_PATH`. I'm not sure of the best way to get this all consistent.

Reply:
May be we should switch to [<PackageName>_ROOT](https://cmake.org/cmake/help/latest/policy/CMP0074.html). But I see what you are saying about reporting paths.